### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by menyagah**
# In this section:
 - I used webpack to build my project files.
 - I created an index.html file and wrote my HTML markup there.
 - I  created an empty To-Do List placeholder <ul> element.
 - created an index.js file and set an array of some simple to-do tasks (array of objects).
 - Wrote a function to iterate over the tasks array and populate the HTML list item element for each task
 - Created a style.css and set rules for the To-Do List
 - Fixed linter errors
 - Fixed tse requested change: base branch from develop to main
 - Below is a live link to the project
           https://fierce-inlet-77592.herokuapp.com/

### CODE REVIEW

**Comment by Lameck1** [comment]
Hi @menyagah, 

While you made a great effort in this project, unfortunately, code review will commence on this PR.

## Invalid Code Review Request

You have submitted a project, that has failed linter checks.

### Suggestion
Lighthouse and Webhint are really not nessesarry for all JS projects especially when we are using Webpack. Feel free to remove them, or maybe just Lighthouse and nobody will ask you thing 😃. All the best.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,147 @@
...
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+<!-- LICENSE -->
+## License
+
+Distributed under the MIT License. See `LICENSE` for more information.
```
Comment (henatan99) [inline README.md:118]: - [ ] Kindly add a link to an MIT license file. 
￼

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @menyagah , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] In this project you are expected to make your `main` branch the base branch when you try to make a pull request for your feature branches. If you want to check back the Github flow requirements, please follow this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md). Kindly make a change to your base branch using the following simple steps:
```
1. Under your repository name, click Pull requests.
2. In the "Pull Requests" list, click the pull request you'd like to modify.
3. Next to the pull request's title, click Edit.
4. In the base branch drop-down menu, select the base branch you'd like to compare changes against.
```
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — COMMENTED** [review]
Hi, 

Good job so far! 👏🏼 @menyagah 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

### Optional suggestion
 - Kindly follow the github flow as requested by the previous reviewer. You would have created develop as a default branch and create another branch called feature for deployment and later you can merge feature to the develop branch instead of main.
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @menyagah 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
As pointed by the previous code reviewer, you would have created two branches: the develop and feature branch for deploying the changes and make develop as a default branch to be merged after getting approved.
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**